xml_inscriptions/transcribed/NJ.Princ.PU.AM.L.2003-128.xml

```diff
@@ -0,0 +1,170 @@
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date notBefore="0200" notAfter="0299">Third centiry CE</date>
+                            <placeName ref="asia.asia_minor">Seleucia, Turkey</placeName>
+                        </origin>
+                        <acquisition>
+                            <!-- subsequent locations if inscription was moved -->
+                            <date when="0001">Date to be displayed</date>
+                            <p>Gift of the Committee for the Excavation of Antioch to Princeton
+                                University</p>
+                        </acquisition>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#"/>
+                        <biblScope unit="page">00</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ref type="external"
+                            target="https://artmuseum.princeton.edu/collections/objects/41955"
+                            >Externanal Reference Princeton Art Museum</ref>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2020-03-17" who="GG">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url=""/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+            <lb n="1"/> N I V
```

Comment:
Just looking at the picture quickly, this kind of looks like `<unclear>u</unclear>niu<unclear>s</unclear>`, no?